packages/mui-base/src/SelectUnstyled/SelectUnstyled.test.tsx

```diff
@@ -96,10 +96,10 @@ describe('<SelectUnstyled />', () => {
         });
 
         const listbox = getByRole('listbox');
-        userEvent.keyPress(listbox, { key });
+        fireEvent.keyDown(listbox, { key });
```

Comment:
`userEvent.keyPress` was problematic as right after `keydown` focus changed and the `keyup` event was fired on another element.